lib/relation-definition.js

```diff
@@ -2458,20 +2458,6 @@ EmbedsMany.prototype.prepareEmbeddedInstance = function(inst) {
         process.nextTick(callback);
       }
     };
-    var originalTrigger = inst.trigger;
-    inst.trigger = function(actionName, work, data, callback) {
-      if (typeof work === 'function') {
-        var originalWork = work;
-        work = function(next) {
-          originalWork.call(this, function(done) {
-            inst.triggerParent(actionName, function(err, inst) {
```

Comment:
Ha, I was not aware of this feature. Could you please find more details about what was this code performing from user's point of view and then describe it in release notes?

My guess is that this code is ensuring that when you call something like `Categories.products.create(...)`, the hooks are triggered both on Category and Product models. That's just a wild guess. `git blame` and related discussions in pull requests may help you to shed more light on this piece of code.